src/main/scala/com/wavesplatform/settings/LegacyConfigTransformer.scala

```diff
@@ -88,6 +88,8 @@ object LegacyConfigTransformer {
     "matcher.host"        -> "matcher.bind-address",
     "matcher.port"        -> "matcher.port",
     "matcher.minOrderFee" -> "matcher.order-match-tx-fee",
+    "matcher.baseAssets"  -> "matcher.base-assets",
```

Comment:
We don't  have any PROD-running nodes with matchers, we do not need legacy-converters